### DISCUSSION TITLE
Intent detection QnA

### DISCUSSION THREAD

**Discussion by doumlegare** [FlowiseAI/Flowise#1625] [Q&A]
Do you use a system,  tool or information into your document to create intent subject for your QnA to retrieve intent for a next step after an answer.
Can we add some parameters key and value in Curl response to get from API? 
thanks

**Comment by rvy474** [comment]
Can you maybe share an example of what you are trying to do. Because its not clear right now. What do you mean by 'intent subject'?

**Reply by doumlegare** [reply]
I have found a solution with chat completion.
The goal is to retrieve the subject of last question asked and redirect to other question where button and choice would be available
